units/INU2003/INU2003_unit.bp

```diff
@@ -141,9 +141,9 @@ UnitBlueprint {
         UniformScale = 0.12,
     },
     Economy = {
-        BuildCostEnergy = 1650,
-        BuildCostMass = 220,
-        BuildTime = 1100,
+        BuildCostEnergy = 2500,
+        BuildCostMass = 300,

```

Comment:
this is a pretty high increase in cost for this unit

Reply:
This is reasonable increace to a unit that shoots 4 missiles at once and can cross water

Reply:
yes, but this should have been part of the original change already imo